game.js

```diff
@@ -547,69 +547,60 @@ module.exports = function createGame(options) {
         return lodash.intersection(state.roles, lodash.flatten([roles]))[0];
     }
 
-    function command(playerIdx, command) {
-        debug('command from player: ' + playerIdx);
-        debug(command);
-        var i, action, message;
-        var playerState = state.players[playerIdx];
-        if (playerState == null) {
-            throw new GameException('Unknown player');
+    const playerIsReady = (command, playerIdx) => {
+        if (state.players[playerIdx].isReady !== true) {
+            throw new GameException('You cannot start the game');
         }
-        if (command.command == 'leave') {
-            // You can always leave, even if your state id is old.
-            playerLeft(playerIdx);
+    };
+    const playerIsUndead = (command, playerIdx) => {
```

Comment:
To me, all these functions would be a lot clearer if they were named, e.g., `checkPlayerIsAlive`, `checkItIsOurTurn`, etc.